install.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# System
+sudo apt install build-essential automake pkg-config libtool libffi-dev libgmp-dev python3-pip
+
+sudo pip3 install pipenv
+pipenv --three
+VENV="$(pipenv --venv)"
+
+# Sol - TODO: integrity
+wget https://github.com/ethereum/solidity/releases/download/v0.4.19/solc-static-linux -O "${VENV}/bin/solc"
```

Comment:
Why not bump the release number from time to time?

Reply:
Yes - looks like were up to V 0.4.20 now.

Reply:
:heavy_check_mark: